tests/README-integration-tests.md

```diff
@@ -0,0 +1,18 @@
+# Integration Tests #
+
+The integration tests are written as Ansible playbooks targetted for all hosts.
+They follow the [Fedora's Standard Test
+Interface](https://fedoraproject.org/wiki/CI/Standard_Test_Interface) and can
+be run by the [Linux Sytem Roles Test
+Harness](https://github.com/linux-system-roles/test-harness). You can also run
+them directly, for example to run them against the [CentOS Cloud
```

Comment:
I will use "steps/points" here, it is hard to follow in this long sentence.

Reply:
I broke it up into two sentences.